sdk/scheduler/src/main/java/com/mesosphere/sdk/api/JettyApiServer.java

```diff
@@ -1,6 +1,8 @@
 package com.mesosphere.sdk.api;
 
+import org.eclipse.jetty.server.NetworkConnector;
 import org.eclipse.jetty.server.Server;
+
```

Comment:
yes, the line break on change in org is the same imposed magical formatting rule as the other line breaks in the import section.